tests/system/env-template.yml

```diff
@@ -50,3 +50,9 @@ MLRUN_SYSTEM_TESTS_SLACK_WEBHOOK_URL:
 
 # sql db path string for test_sql_db - e.g. mysql+pymysql://<username>:<password>@<host>:<port>/<db_name>
 MLRUN_SQL__URL:
+
+# AWS bucket name
+AWS_BUCKET_NAME: 
+# Access key parameters as created by AWS IAM secret key mechanism
```

Comment:
I don't think we need to explain it here either
```suggestion
```

Reply:
will fix